Remove the -pedantic and -Werror flags from the CPPFLAGS.

Otherwise the compiler will complain and exit due to extra semicolons
introduced by the  Boost 1.73.0 dependency.

diff --git a/Makefile.am b/Makefile.am
index 3140b1e..ba0c360 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -43,9 +43,7 @@ test_readpdb_LDADD	=	$(shared_LDADD) \
 									$(BOOST_UNIT_TEST_FRAMEWORK_LIB)
 
 AM_CPPFLAGS	=	-std=c++11 \
-							-pedantic \
 							-Wall \
-							-Werror \
 							-Wno-reorder \
 							$(BOOST_CPPFLAGS) \
 							$(BZ2LIB_CPPFLAGS) \
